config/jsdoc/api/template/static/styles/jaguar.css

```diff
@@ -2,11 +2,24 @@
 .navbar-inverse {
   border: 0;
 }
-.navbar-inverse .navbar-inner {
+.navbar{
   background: #1F6B75;
   height: 50px;
   text-shadow: 1px 1px 2px rgba(0,0,0,0.25);
 }
+
+.navbar-brand img {
+height: 35px;
+vertical-align: middle;
+margin-right: 5px;
+display: inline-block;
+}
+
+
+
+
+
+/*
```

Comment:
When you're happy with the final result, please delete any commented style blocks (commented code is only a source of mystery to future devs).

Reply:
yes sure i will do  ! but for now i not stable for api header i have to work again with those styles ...